ethereum/hybrid_casper/validator.py

```diff
@@ -68,8 +68,8 @@ def epoch_blockhash(self, state, epoch):
 
     def get_recommended_casper_msg_contents(self, casper, validator_index):
         return \
-            casper.get_recommended_target_hash(), casper.get_current_epoch(), \
-            casper.get_recommended_source_epoch()
+            casper.recommended_target_hash(), casper.current_epoch(), \
```

Comment:
now that "get_" is removed from each call. The line is 98 chars (less than 100) and should probably just go on one line:

```python
return casper.recommended_target_hash(), casper.current_epoch(), casper.recommended_source_epoch()
```